src/components/OrgPicker/OrgPicker.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * While visually similar to the USWDS Combo box component, this expandable element presents a scrollable list of links. The link at the bottom of the menu directs the user to a page listing all the organizations they can access.
+ */
+import React from 'react';
+// import { OrgObj } from '@/api/cf/cloudfoundry-types';
+import { Button } from '@/components/uswds/Button';
+import { useState } from 'react';
+import Image from 'next/image';
+import collapseIcon from '@/../public/img/uswds/usa-icons/expand_more.svg';
+import { OrgPickerList } from './OrgPickerList';
+import { OrgPickerFooter } from './OrgPickerFooter';
+
+export function OrgPicker({ single }: { single: Boolean }) {
+  const [isOpen, setIsOpen] = useState(false);
+  const togglePicker = () => setIsOpen(!isOpen);
+
+  return !single ? (
+    <div className="display-block desktop:display-flex desktop:position-absolute">
+      <span className="usa-label font-body-2xs padding-right-105">
```

Comment:
[non-blocking; a11y question] Would it be worth adding an `id` to this, and slapping an `aria-describedby` attribute that points to this element?

(Honest question, I don’t know if that works well for this interaction model!)

Reply:
(Comment withdrawn, after looking over #518 I don’t think that’d be right here!)